GME/Gme/GMEView.cpp

```diff
@@ -468,6 +468,7 @@ BEGIN_MESSAGE_MAP(CGMEView, CScrollZoomView)
 	ON_UPDATE_COMMAND_UI(ID_FILE_CHECK, OnUpdateFileCheck)
 	ON_UPDATE_COMMAND_UI(ID_FILE_CHECKSEL, OnUpdateFileCheckSelected)
 	ON_UPDATE_COMMAND_UI(ID_FILE_SETTINGS, OnUpdateFileSettings)
+	ON_UPDATE_COMMAND_UI(ID_HELP_CONTENTS, OnUpdateFileSettings)

```

Comment:
Is this necessary?

Reply:
Yes, That is what triggers the update to the help menu lists

Reply:
If I comment it out, it still works. Is only the entry in MainForm needed?

Reply:
You're right. 
I thought it was needed because the toolbar is different when a part/component/testbench is open, which I thought was only controlled by GMEView.